tests/test_wave_01.py

```diff
@@ -158,13 +157,14 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
```

Comment:
👍 

Consider also confirming the genre and rating are as expected by adding assertions for those values too.